src/renderer/src/components/exchange/NewExchangeTemplate.tsx

```
@@ -223,19 +244,21 @@ const NewExchangeTemplate = (): ReactElement => {
+        <FormLabel>{t('Instructions On Complete')}</FormLabel>
         <Textarea
           value={participantInstructionsOnComplete}
           onChange={handleChangeParticipantInstructionsOnComplete}
         />
         <FormHelperText>
-          These are the instructions that are shown to the participant when the
-          exchange has been marked as completed.
+          {t(
+            'These are the instructions that are shown to the participant when the exchange has been marked as completed.',
+          )}
+          '
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)